db/data_migration/20160512150912_republish_corporate_information_pages.rb

```diff
@@ -0,0 +1,7 @@
+CorporateInformationPage
+  .published
+  .includes(:document)
+  .(:document, :translations)
```

Comment:
Here I was thinking it's one of those syntactic sugar things that's impossible to Google for. :D